### PR TITLE
New Oxy Script

### PR DESCRIPTION
**Pull request by Zeeroqt**
- Buffed Drug Sale Prices
- Added Oxy to drug selling
- Added a chance of getting oxy in robbing houses
- Added a Big Reward Chance
- Added the ability to sell the Big Reward
- Added Oxy as a Healing Item
- Added consuming pill animation 
- Added Overall Oxy Script

### CODE REVIEW

**Comment by minipunch** [comment]
Can you explain a bit how it works for me please

**Comment by Zeeroqt** [comment]
> Can you explain a bit how it works for me please

Yeah so basically u go up to a local and u have to pay a fee to start this mission. U get a vehicle to start doing these oxy runs. After 30 seconds to a minute u while receive a blip on ur map to go to. After going to the location there will be a ped that spawns in and basically do a trade deal with an animation. U can mainly get money but have a 25% chance getting oxy. And u have a 3.5 chance of getting a rare painting that can sell for $$$

**Comment by Zeeroqt** [comment]
Could u show a clip of it giving u a medkit animation. It shouldn’t, it tested it out and only got the pill animation

**Comment by Zeeroqt** [comment]
It takes 35 seconds to get the full 70 health from it. Remember u only get a % chance of getting it. U don’t just get it every run.

**Comment by ChangyBoi** [comment]
This was tested when you first made the PR and I don't have a clip

Theres no medkit animation its just the pill animation

The way we saw it was, Chris got low hp, pop the oxy, health starts to regen and after a couple of seconds, it will boost your health in one big swoop. If that makes sense lol

**Comment by Zeeroqt** [comment]
🤔

**Review by DatBoiWeep — CHANGES_REQUESTED** [review]
Double check the healing. Either lower the initial healing or remove the part during the animation imo.

File: resources/interaction-menu/cl_interaction.lua [hunk 1]
```diff
@@ -1976,6 +1985,18 @@ function playHealingAnimation(ped)
...
 end
 
+function playOxyHealingAnimation(ped)
+	exports.globals:loadAnimDict("mp_suicide")
+	TaskPlayAnim(ped, "mp_suicide", "pill", 8.0, 8.0, 1.0, 48, -1, 0, 0, 0)
+	local count = 70
+    while count > 0 do
+        Citizen.Wait(85)
+        count = count - 2
+        SetEntityHealth(PlayerPedId(), GetEntityHealth(PlayerPedId()) + 1)
```
[the remark this answers is not in the document]
Comment (Zeeroqt) [reply resources/interaction-menu/cl_interaction.lua:1995]: hmm ok thanks

**Comment by Zeeroqt** [comment]
@minipunch lookin good?

File: resources/interaction-menu/cl_interaction.lua (hunk 1 above)
[the remark this answers is not in the document]
Comment (Zeeroqt) [reply resources/interaction-menu/cl_interaction.lua:1994]: fixed forgot to respond to

**Comment by minipunch** [comment]
Can u upload a video of how to do it so I can test it

and fix the merge conflict plz

**Pull request closed without merging**